### ISSUE TITLE
Submission method 'jQuery.ajaxError' title is blank

### ISSUE THREAD

**Issue by theit8514** [exceptionless/Exceptionless.JavaScript#26]
When an exception is handled via jQuery.ajaxError, the title in the Exceptionless UI is blank.

[jQuery.ajaxError](http://api.jquery.com/ajaxerror/) passes the error string as the 4th argument, which is [being passed](https://github.com/exceptionless/Exceptionless.JavaScript/blob/41e9559ae4725f53e65a8f38a0a71035f6c86107/src/exceptionless.ts#L77) to processUnhandledException which accepts Error objects. The result is that the error (e.g. "Internal Server Error") is not parsed correctly by TraceKit.

Outputs JSON data as a result:

``` json
[{"date":"2015-09-23T21:25:05.685Z","type":"error","source":"api/Some/ApiCall?id=10","data":{"status":500,"response":"{\"Message\":\"An error has occurred.\"}","@error":{"stack_trace":[]},"@request":{"user_agent":"Mozilla/5.0 (Windows NT 6.3; WOW64; rv:40.0) Gecko/20100101 Firefox/40.0","is_secure":true,"host":"www.example.com","port":80,"path":"/","cookies":{"Cookie":""},"query_string":null,"referrer":"https://www.example.com"},"@submission_method":"JQuery.ajaxError"},"tags":["Example","JavaScript"]}]
```

**Comment by niemyjski** [comment]
Thanks for reporting this. I'm blocked on this by a second issue regarding tracekit that I need to track down. In the mean time I may comment out this code in the next minor release.

**Comment by niemyjski** [comment]
This was fixed in 2.0 of the client. It won't create a message if one isn't defined in the error. But it will log it without throwing. I've added this to the examples.
